requirements.txt

```diff
@@ -1,3 +1,4 @@
 lir==0.1.23
 lrbenchmark>=0.1.1
 numpy==1.24.3
+scikit-learn==1.2.2
```

Comment:
Given fix in lr-benchmark, this becomes obsolete, I'd think.

Reply:
denk het ook idd